test/functional/fixtures/native-dialogs-handling/test.js

```diff
@@ -0,0 +1,270 @@
+        });
+
+        it('Should fail when the unexpected confirm dialog appears after redirect in iframe', function () {
+            return runTests('testcafe-fixtures/in-iframe.test.js', 'Unexpected confirm after redirect in iframe - should fail', { shouldFail: true })
+                .catch(function (err) {
+                    var expectedError = [
+                        'Error at step "1.Click link "confirm page"":',
+                        'Unexpected system confirm dialog Confirm appeared.'
+                    ].join(' ');
+
+                    expect(err).eql(expectedError);
+                });
+        });
+
+        it("Should fail if the expected confirm dialog doesn't appear after an action in iframe", function () {
+            return runTests('testcafe-fixtures/in-iframe.test.js', 'No expected confirm after action in iframe - should fail', { shouldFail: true })
+                .catch(function (err) {
+                    var expectedError = [
+                        'Error at step "1.Click submit button "Button"":',
+                        'The expected system confirm dialog did not appear.'
+                    ].join(' ');
+
+                    expect(err).eql(expectedError);
+                });
+        });
+
+        it("Should fail if the expected confirm dialog doesn't appear after redirect in iframe", function () {
+            return runTests('testcafe-fixtures/in-iframe.test.js', 'No expected confirm after redirect in iframe - should fail', { shouldFail: true })
+                .catch(function (err) {
+                    var expectedError = [
+                        'Error at step "1.Click link "This page"":',
+                        'The expected system confirm dialog did not appear.'
+                    ].join(' ');
+
+                    expect(err).eql(expectedError);
+                });
+        });
+    });
+
+    describe('Alert', function () {
+        it('Should fail if the unexpected alert dialog appears after click on the fake link', function () {
+            return runTests('testcafe-fixtures/index.test.js', 'Unexpected alert after click on the link without redirect - should fail', { shouldFail: true })
+                .catch(function (err) {
+                    var expectedError = [
+                        'Error at step "1.Click alert button":',
+                        'Unexpected system alert dialog  appeared.'
+                    ].join(' ');
+
+                    expect(err).eql(expectedError);
+                });
+        });
+
+        it('Should fail if the unexpected alert dialog appears after an action', function () {
+            return runTests('testcafe-fixtures/index.test.js', 'Unexpected alert after action - should fail')
+                .catch(function (err) {
+                    var expectedError = [
+                        'Error at step "1.Click alert button":',
+                        'Unexpected system alert dialog alert button clicked appeared.'

```

Comment:
oh, I managed to read it - are there quotes that are missing?
`Unexpected system alert dialog \'alert button clicked\' appeared.`